Formula/o/openslp.rb

```diff
@@ -15,18 +15,39 @@ class Openslp < Formula
     sha256 x86_64_linux: "aa1988503f1e9688dfd80e0331392ab29a053e62197b60653e933ee1bc681efb"
   end
 
-  on_macos do
-    depends_on arch: :x86_64
-  end
-
   # Fix -flat_namespace being used on Big Sur and later.
   patch do
     url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-pre-0.4.2.418-big_sur.diff"
     sha256 "83af02f2aa2b746bb7225872cab29a253264be49db0ecebb12f841562d9a2923"
   end
 
   def install
+    # Workaround for arm64 macOS to use fallback global mutex as USE_APPLE_ATOMICS
+    # condition uses deprecated functions and code doesn't compile
+    inreplace "common/slp_atomic.c", <<~C, "#else\n" if OS.mac? && Hardware::CPU.arm?
+      #elif defined(__APPLE__)
+      # define USE_APPLE_ATOMICS
+      #else
+    C
```

Comment:
maybe upstream the change or log a ticket for it?

Reply:
Looks like development has moved to github: https://github.com/openslp-org/openslp

Reply:
Opened upstream issue:
- https://github.com/openslp-org/openslp/issues/19